ansible/roles/kafka/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+apache_mirror: apache.mirror.anlx.net
+kafka_version: 0.8.2.2
+kafka_user: kafka
+kafka_group: kafka
+kafka_data_dir: /opt/data/kafka/data
+kafka_conf_dir: /etc/kafka/config
+kafka_log_dir: /var/log/kafka
```

Comment:
My general model has been that logs that will output "commcareish errors" would go in `log_home`, things like pillowtop errors.  I leave things in `/var/log` that are more system errors, e.g. `/var/log/nginx/error.log` has errors when nginx cant start, or the process ends unexpectedly.